ios/ci_scripts/ci_post_clone.sh

```
@@ -0,0 +1,28 @@
+# Install Flutter using git.
+git clone https://github.com/flutter/flutter.git --depth 1 -b stable $HOME/flutter
+export PATH="$PATH:$HOME/flutter/bin"
+
+# Install Flutter artifacts for iOS (--ios), or macOS (--macos) platforms.
+flutter precache --ios
+
+# Install Flutter dependencies.
+flutter pub get
+
+# Install CocoaPods using Homebrew.
+HOMEBREW_NO_AUTO_UPDATE=1 # disable homebrew's automatic updates.
```

Shellcheck

[warning] HOMEBREW_NO_AUTO_UPDATE appears unused. Verify use (or export if used externally).

(SC2034)